writing-guide/readme.md

```diff
@@ -0,0 +1,77 @@
+> **Goal of this guide:** To help team members write in a clear manner
+
+# How to write well
+
+## Explain the content of the article
+
+People are trained to filter information very efficiently and are always thinking about how to best spend their valuable time. We want to know beforehand why we spend time on an activity. You should, therefore, grab the attention of the reader very early on. Only once interest has been piqued can you be sure that the reader will want to devote attention to the rest of the article, which will lead to a relevant reading experience.
+
```

Comment:
I agree. Tess and I actually talked about this before making a pull request but were both too brain fried to change it at that moment. It's actually a really good indication of your baller reviewing skills that you caught it.